server/src/main/java/org/apache/iotdb/db/metadata/MetadataOperationType.java

```diff
@@ -34,5 +34,6 @@ private MetadataOperationType(){
   public static final String UNLINK_MNODE_FROM_PTREE = "7";
   public static final String ADD_INDEX_TO_PATH = "8";
   public static final String DELETE_INDEX_FROM_PATH = "9";
-  public static final String DELETE_STORAGE_GROUP_FROM_MTREE = "10";
+  public static final String SET_TTL = "10";
+  public static final String DELETE_STORAGE_GROUP_FROM_MTREE = "11";
```

Comment:
Why not directly add a field SET_TTL = "11"?

Reply:
Because when I created the field SET_TTL, there is no such field DELETE_STORAGE_GROUP_FROM_MTREE. And later I merged master which brought DELETE_STORAGE_GROUP_FROM_MTREE.